### PR TITLE
Adding quiet parameter

### PR DESCRIPTION
**Pull request by arkadiuszbeer**
Task https://github.com/insightsengineering/staged-dependencies-action/issues/20

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -85,3 +89,5 @@ runs:
         SD_ENABLE_CHECK: "${{ inputs.enable-check }}"
         SD_RENV_RESTORE: "${{ inputs.renv-restore }}"
         SD_GIT_REF: "${{ inputs.git-ref }}"
+        SD_QUIET: "${{ inputs.quiet }}"
+
```
Comment (cicdguy) [inline action.yml:93]:
```suggestion
```

File: staged-dependencies.R [hunk 2]
```diff
@@ -56,7 +60,7 @@ if (threads == "auto") {
 }
 
 # Install the remotes package
-if (!require("remotes", quietly = TRUE)) {
+if (!require("remotes", quietly = sd_quiet)) {
```
Comment (cicdguy) [inline staged-dependencies.R:63]: This won't work. You need type conversion here.

**Review by cicdguy — CHANGES_REQUESTED** [review]

File: staged-dependencies.R (hunk 2 above)
Comment (arkadiuszbeer) [reply staged-dependencies.R:63]: yes, I have it in line 41
`sd_quiet <- isTRUE(as.logical(Sys.getenv("SD_QUIET", "true")))`

**Review by cicdguy — APPROVED** [review]

**Comment by cicdguy** [comment]
Please also update the README with info about the new input parameter.

File: action.yml [hunk 3]
```diff
@@ -85,3 +89,4 @@ runs:
         SD_ENABLE_CHECK: "${{ inputs.enable-check }}"
         SD_RENV_RESTORE: "${{ inputs.renv-restore }}"
         SD_GIT_REF: "${{ inputs.git-ref }}"
+        SD_QUIET: "${{ inputs.quiet }}"
```
Comment (cicdguy) [inline action.yml:92]:
```suggestion
        SD_QUIET: "${{ inputs.quiet }}"

```

**Pull request merged**